src/AppScopeTrait.php

```diff
@@ -63,7 +63,7 @@ trait AppScopeTrait
      */
     public $unique_hashes = [];
 
-    private function assertInstanceOfApp(object $app): void
+    protected function assertInstanceOfApp(object $app): void
```

Comment:
https://3v4l.org/2qPLn :D

please try itself before

Reply:
i need to override it not use it as it is, did you know what are you doing?